README.md

```diff
@@ -24,14 +24,26 @@ The `deepstack-ui` is designed to be run in a docker container. The UI picks up
 - DEEPSTACK_UI_DEBUG_MODE : options `True` or `False` (default). Lowers the minimum confidence threshold to 1%
 ```
 
-From the root dir, build the deepstack-ui container from source and then run the UI, passing the `DEEPSTACK_IP` environment variable:
+From the root dir, either build the deepstack-ui container from source or pull from Docker Hub: and 
+
+For x86-64 architecture
 ```
-    docker build -t deepstack-ui .
+    docker build -t deepstack-ui . -f Dockerfile
     OR
     docker pull robmarkcole/deepstack-ui:latest
+```
+
+Or for arm64 architecture (e.g. Raspberry Pi, [Apple M1 chips](https://docs.docker.com/docker-for-mac/apple-m1/))
```

Comment:
It is for rpi only

Reply:
Sorry, was fighting with pyarrow all day yesterday on different OS's. Seems to be a big issue with it and pip on anything other than amd64. Overnight I was finally able to build a multi-arch Ubuntu base image with it preinstalled by building it from source. Hopefully it will eliminate the need for two dockerfiles, too. I'm going to test it this morning, so crossing my fingers...